tests/enable-clustering.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from "./fixtures/lxd-test";
+import { skipIfNotSupported } from "./helpers/cluster";
+import { gotoURL } from "./helpers/navigate";
+
+test("check enabling clustering", async ({ page, lxdVersion }) => {
+  skipIfNotSupported(lxdVersion);
```

Comment:
this test should only run in the `enable-clustering` suites. Similar to the login spec test.